samples/mixed/provider/src/main.rs

```diff
@@ -182,22 +200,43 @@ async fn start_vehicle_simulator(
             }
 
             info!("Publishing the values: Ambient air temperature is {ambient_air_temperature}; Is air conditioning active is {is_air_conditioning_active}; Hybrid battery remaining is {hybrid_battery_remaining}");
+            let ambient_air_temperature_property: AmbientAirTemperatureProperty =
```

Comment:
the type annotation here and in the two following let statements are not necessary

Reply:
I prefer it to be explicit.  Leaving as is.